demo-setup/start.sh

```diff
@@ -0,0 +1,14 @@
+# Script assumes that host system has go-lang installed
+# The script assumes that openfinex opens it's port on 8001
```

Comment:
This sripts assumes to have the openfinex components somewhere.
So I'd suggest to either:
1) Add intsructions where to get these files from and/or where to place them manually
2) add a script option somewhere

Reply:
+ compose folder from openfinex

Reply:
I have added the compose folder, as we discussed the openfinex can be shared internally